charts/Makefile

```diff
@@ -0,0 +1,44 @@
+HELM ?= helm
+HELM_DOCS_IMAGE ?= jnorwood/helm-docs:v1.11.0
+BUILD_DIR ?= bin
+OUTPUT_DIR ?= $(BUILD_DIR)
+CHART_DIR := $(BUILD_DIR)/gadget
+
+IMAGE_TAG := $(shell git describe --tags --abbrev=0 --match "v*")
+# CHART_VERSION must be a valid SemVer string so strip the leading 'v'
+CHART_VERSION := $(IMAGE_TAG:v%=%)
+APP_VERSION ?= $(IMAGE_TAG)
+
+clean:
+	rm -rf $(BUILD_DIR)
+
+build: clean
+	@echo "Building chart:"
+	mkdir -p $(BUILD_DIR)
+	cp -r gadget $(BUILD_DIR)
+	cp -r ../pkg/resources/crd/bases $(CHART_DIR)/crds
+	mv $(CHART_DIR)/Chart.yaml.tmpl $(CHART_DIR)/Chart.yaml
+	sed -i "s/%VERSION%/$(CHART_VERSION)/g" $(CHART_DIR)/Chart.yaml
+	sed -i "s/%APP_VERSION%/$(APP_VERSION)/g" $(CHART_DIR)/Chart.yaml
+	@echo "Preparing docs"
+	docker run --user $(shell id -u):$(shell id -g) -v $(shell pwd)/$(CHART_DIR):/helm-docs $(HELM_DOCS_IMAGE) -s file
+	@echo "Charts available at: $(CHART_DIR)"
+
+# install uses 'helm upgrade --install' to make chart installation idempotent,
+# also helm upgrade does not update CRDs so we need to apply them separately
```

Comment:
:( this is too bad as users will have to handle additional commands manually. 

According to https://github.com/helm/helm/issues/8668#issuecomment-683925646:

> If you're chart doesn't install custom resources just the CRDs... you can put the CRDs right into the templates directory.

That seems to be our case. Perhaps we can investigate more to see if we can improve this. Anyway, I'll agree this an improvement and not a blocker for this PR.